README.md

```diff
@@ -57,9 +57,21 @@ Refer [concepts](doc/CONCEPTS.md) to understand the concepts referred to in this
   - Kafka on localhost:9092
   - ZooKeeper on localhost:2181
   - RabbitMQ on localhost:5672
-
 - Run tests: `make test`
 
+#### Running a cluster set up locally
+
+- `make setup-cluster` This starts
+    - 3 Kafka brokers on localhost:9091, localhost:9092 and localhost:9093
+    - Zookeeper on localhost:2181
+    - RabbitMQ on localhost:5672
+
+#### Running tests via a cluster
+- Rename the file config.test.cluster.edn to config.test.edn
```

Comment:
Why can't this be made the default? Instead of the single instance setup.
> We are going to keep both as of now, later remove. 

First, there shouldn't be any hardcoding for the test file name.
This manual step shouldn't be needed.

Reply:
If above is not possible, 
mv step first, 
gco step second.

Reply:
Manual can be kept as last resort.